clients/web/src/lib/components/abstract/Chatwoot/Chatwoot.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+    import {onMount} from "svelte";
+    import {browser} from "$app/env";
+    import {session} from "$app/stores";
+
+    import {chatwootSettings} from "./Chatwoot.constants";
+
+    const {
+        enabled, url, websiteToken,
+    } = $session.config.chatwoot;
+
+    let identifier: string;
+    let hash: string;
+
+    onMount(async () => {
+        if (browser && enabled) {
+            // Append protocol to url
+            const _url = url.startsWith("http") ? url : `https://${url}`;
+
+            // Fetch identifier and HMAC hash
+            const res = await fetch("/auth/chatwoot");
+            if (!res.ok) return;
+            ({identifier, hash} = await res.json());
```

Comment:
Would it be feasible to put this into a load function?

Reply:
https://kit.svelte.dev/docs/loading
> A component that defines a page or a layout can export a load function that runs before the component is created. 

`load` can only be used in pages/layouts